benches/benches/vm_set/blockchain.rs

```diff
@@ -441,6 +483,33 @@ pub fn run(c: &mut Criterion) {
 
     csiz.finish();
 
+    let mut bsiz = c.benchmark_group("bsiz");
```

Comment:
How were we getting the values for `bldd` and `bsiz` before? It looks like they didn't exist before.

Reply:
We had a task to add benchamrks for these upcodes: https://github.com/FuelLabs/fuel-core/issues/2049

Before we just copied values from similar contract opcodes